cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetRDKafka.cmake

```diff
@@ -0,0 +1,22 @@
+find_path(RDKAFKA_INCLUDE "librdkafka" HINTS "$ENV{RDKAFKA_ROOT}/include")
+find_library(RDKAFKA++_LIBRARY "rdkafka++" HINTS "$ENV{RDKAFKA_ROOT}/lib" "$ENV{RDKAFKA_ROOT}/build")
+
+# function(find_and_configure_rdkafka VERSION)
```

Comment:
Should this be addressed before merging?

Reply:
librdkafka doesn't seem to be a well enough formatted CMake project to build via CPM. I'm going to leave this file as a stub in case that changes in the near future, I know they are working on it, and just leave the find_package commands in there.

Reply:
Unresolving for discussion. Is there value in us at least creating a target to use based on these variables instead of using them directly? cc @robertmaynard

Reply:
We can solve this in a follow up.

Reply:
A target would improve the experience when either of these finds fail, as we currently we have no error handling.
Moving to a model where we create a target when both have succeeded would allow easier downstream checks.

```cmake
if(RDKAFKA_INCLUDE AND RDKAFKA++_LIBRARY)
  add_target( rdkafka INTERFACE)
  target_link_libraries(rdkafka INTERFACE "${RDKAFKA++_LIBRARY}")
  target_include_directories(rdkafka INTERFACE "${RDKAFKA_INCLUDE}")

  add_target( RDKAFKA::RDKAFKA ALIAS rdkafka)
endif()
```

Reply:
@jdye64 want to give this a shot?